step-02/index.html

```diff
@@ -22,9 +18,7 @@ <h1>Realtime communication with WebRTC</h1>
     <button id="hangupButton">Hang Up</button>
   </div>
 
-  <script src="js/lib/adapter.js"></script>
+  <script src="//webrtc.github.io/adapter/adapter-latest.js"></script>
```

Comment:
Protocol relative URLs are now discouraged: https://paulirish.com/2010/the-protocol-relative-url

Reply:
Done.